README.md

```diff
@@ -0,0 +1,100 @@
+# sql_insert_writer
+
+[![PyPI Status](https://img.shields.io/pypi/v/sql_insert_writer.svg)](https://pypi.python.org/pypi/sql_insert_writer)
+[![CircleCI](https://circleci.com/gh/18F/sql_insert_writer.svg?style=svg)](https://circleci.com/gh/18F/sql_insert_writer)
+[![Code Climate](https://codeclimate.com/github/18F/sql_insert_writer/badges/gpa.svg)](https://codeclimate.com/github/18F/sql_insert_writer)
+[![Test Coverage](https://codeclimate.com/github/18F/sql_insert_writer/badges/coverage.svg)](https://codeclimate.com/github/18F/sql_insert_writer/coverage)
+[![Dependency Status](https://gemnasium.com/badges/github.com/18F/sql_insert_writer.svg)](https://gemnasium.com/github.com/18F/sql_insert_writer)
+
+Helps generate highly readable SQL INSERT statements
+
+```
+$ sql_insert_writer pet
+
+INSERT INTO pet (
+  id,
+  name,
+  species_name,
+  planet,
+  kg
+)
+VALUES
+(
+  DEFAULT,  -- ==> id
+  DEFAULT,  -- ==> name
+  DEFAULT,  -- ==> species_name
+  DEFAULT,  -- ==> planet
+  DEFAULT  -- ==> kg
+)
+
+$ sql_insert_writer pet animal
+
+INSERT INTO pet (
+  id,
+  name,
+  species_name,
+  planet,
+  kg
+)
+SELECT
+  id,  -- ==> id
+  name,  -- ==> name
+  species_name,  -- ==> species_name
+  planet,  -- ==> planet
+  DEFAULT  -- ==> kg
+FROM animal
+```
```

Comment:
I might be lacking context, or not the right kind of audience for this package, but I'm unclear on a few things here. ~~Where are these column names of `pet` (e.g. `species_name`, `planet`) coming from? Is `sql_insert_writer` introspecting into a database table, or is it getting them via some sort of ORM model defined somewhere in a project?  If it's introspecting into a db table, how is that ...~~

Oh wait, now I'm seeing that a lot of my questions are answered in `docs/usage.rst`. I wonder if it might be best to just have a very minimal `README.md` that simply links out to the documentation, kind of like how [cg-django-uaa](https://github.com/18F/cg-django-uaa#readme) does?  (Since the readthedocs.io link is 404'ing right now, you could potentially just link to `docs/usage.rst` directly, since GitHub renders RST.)

Reply:
I'll make the link to usage.rst more prominent and earlier in the doc, and for good measure add some text explaining this.  Thanks.